kolibri/core/assets/src/vue/nav-bar/index.vue

```diff
@@ -154,6 +184,9 @@
       footerMsg() {
         return this.$tr('poweredBy', { version: __version }); // eslint-disable-line no-undef
       },
+      footerMsg2() {
+        return this.$tr('poweredBy', { version: __version }); // eslint-disable-line no-undef
```

Comment:
`footerMsg` and `footerMsg2` are the same, what's the use case for `footerMsg2`?